### PR TITLE
255 - Mobile Ticketing

### PR DESCRIPTION
**Pull request by 31Nathan**
WIP

### CODE REVIEW

File: Server/prisma/schema.prisma [hunk 1]
```diff
@@ -60,21 +60,22 @@ model Organization {
...
     organizationId Int?
-    confirmed      Boolean       @default(false)
-    createdAt      DateTime      @default(now())
+    confirmed      Boolean          @default(false)
+    createdAt      DateTime         @default(now())
+    tickets_Mobile Tickets_Mobile[] // ONLY FOR ADMIN ON WEB
 }
 
-model Feedbacks_Web {
+model Feedbsks_Web {
```
Comment (H-chauvet) [inline Server/prisma/schema.prisma:78]: Wrong value

**Review by H-chauvet — CHANGES_REQUESTED** [review]
Be careful with typo

File: Server/prisma/schema.prisma [hunk 2]
```diff
@@ -147,3 +149,14 @@ model Location_Mobile {
...
     ended     Boolean     @default(false)
 }
+
+model Tickets_Mobile {
+    id         Int         @id @default(autoincrement())
+    content    String
+    creator    User_Mobile @relation(fields: [creatorId], references: [id])
+    creatorId  String
+    assigned   User_Web    @relation(fields: [assignedId], references: [uuid])
+    assignedId String
```
Comment (invpuppet) [inline Server/prisma/schema.prisma:159]: the user id for the web is an int so it should have the type

File: Server/prisma/schema.prisma [hunk 3]
```diff
@@ -125,6 +126,7 @@ model User_Mobile {
     Locations        Location_Mobile[]
     Notifications    Notifications_Mobile? @relation(fields: [notificationsId], references: [id])
     notificationsId  String?
+    tickets_Mobile   Tickets_Mobile[]
```
Comment (invpuppet) [inline Server/prisma/schema.prisma:129]: name it in camelCase

**Review by invpuppet — CHANGES_REQUESTED** [review]

**Pull request closed without merging**